daml-lf/data/src/main/2.13/com/daml/lf/data/LawlessTraversals.scala

```diff
@@ -7,10 +7,11 @@ import scala.annotation.tailrec
 import scala.collection.BuildFrom
 
 private[daml] object LawlessTraversals {
-  implicit final class `Lawless iterable traversal`[A](private val seq: Iterable[A])
+  implicit final class `Lawless iterable traversal`[A, This[X] <: Iterable[X]](
+      private val seq: This[A])
```

Comment:
```suggestion
  implicit final class `Lawless iterable traversal`[A, This <: Iterable[A]](
      private val seq: This with Iterable[A])
```

The duplication is so inference goes cleanly.